terraform/templates/ec2_user_data.sh

```diff
@@ -36,12 +36,12 @@ EOF
 
 {% if enable_logrotate %}
 cat > /etc/logrotate.d/libra <<EOF
-${log_path} {
-	daily
-	size 100M
+${host_log_path} {
+	size 500M
```

Comment:
Not sure if half of a gig would cause perf hit.  Something to think about or tweak after deployment.